src/ManageCourses.Api/Services/UnknownMcUserException.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace GovUk.Education.ManageCourses.Api.Services
```

Comment:
wrong namespace
reject

Reply:
`UnknownMcUserException` translate to Unknown McUser Exception

Maybe McUserException and use the message

otherwise there is going be a lot of exception type.

Reply:
McUserNotFoundException